src/no_deps.rs

```diff
@@ -0,0 +1,188 @@
+/**
+ *
+ * This module contains stubs for third-party dependencies of the Magewell Capture SDK.
+ * The SDK requires these dependencies to be present in the build. However, none of them are
+ * necessary to perform capture from PCIe devices, so for a more minimal build, we can eliminate
+ * the dependencies by stubbing out the essential symbols of these libraries.
+ *
+ * It is impossible for the vast majority of these stubs to be called, but we make our best effort
+ * to always return a valid value or error code anyway.
```

Comment:
Pretty much the only thing that gets called during the unit test is `libusb_get_device_list`, which just returns an empty list.